package.json

```diff
@@ -4,12 +4,10 @@
   "description": "Adds the possibility to add recuder asynchronously to your redux store",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/francisprovost/async-reducer.git"
+    "build": "rm -rf dist && npx babel index.js -d dist",
```

Comment:
don't use `npx` if you don't specify the minimum npm version

Reply:
use `rimraf` instead of `rm -rf`